src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +32,32 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #{id}
             </div>
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
+              type="button"
+              className="delete"
+              data-cy="modal-close"
+              onClick={() => onClose(null)}
+            />
```

Comment:
The naming convention for your event handler function `onClose` is not consistent with the recommended conventions. Consider renaming it to `handleClose` to make it clear that it's a function that's meant to handle an event. Here are some [naming conventions](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) for your reference.